src/components/info-list.jsx

```diff
@@ -10,31 +11,19 @@ const InfoList = ({
   highlightFirstItem,
   noTitles,
```

Comment:
Same thing here, add `className` and `...props`

Reply:
done!